src/components/Tokens/TokenDetails/About.tsx

```diff
@@ -1,6 +1,6 @@
 import { Trans } from '@lingui/macro'
-import { SupportedChainId } from '@uniswap/sdk-core'
 import { getChainInfo } from 'constants/chainInfo'
+import { SupportedChainId } from 'constants/chains'
```

Comment:
so we're converging on the `interface` version of this enum? 

i thought using the updated SDK version of the enum was a goal of the upgrade? 

just checking

Reply:
we might do that but right now we don't support arbitrum goerli yet and we should have a team discussion on that. the goal is to just remove chains that aren't supported by permit2

Reply:
for now we can't use the sdk-core version bc that would require us to support BNB and aribitrum goerli